skeleton/docs/default.nix

```diff
@@ -0,0 +1,45 @@
+{ iohkSkeletonPackages ? import ./.. {}
+, pkgs ? iohkSkeletonPackages.pkgs
+}:
+with pkgs;
+
+{
+  example = stdenv.mkDerivation {
+    name = "iohk-skeleton-docs";
+    buildInputs = [ (texlive.combine {
+                      inherit (texlive)
+                        scheme-small
+
+                        # TODO: replace with your own LaTeX package dependencies
+
+                        # libraries
+                        stmaryrd lm-math amsmath extarrows cleveref semantic xcolor appendix
+
+                        # bclogo and dependencies
+                        bclogo mdframed xkeyval etoolbox needspace pgf
+
+                        # font libraries `mathpazo` seems to depend on palatino
+                        # , but it isn't pulled.
+                        mathpazo palatino microtype
+
+                        # libraries for marginal notes
+                        xargs todonotes
+
+                        # build tools
+                        latexmk
+                        ;
+                    })
+                  ];
+    src = ./.;
```

Comment:
I recommend providing a latex source filter, I have one that we use for plutus: https://github.com/input-output-hk/plutus/blob/master/nix/latex.nix#L37